presto-docs/src/main/sphinx/release/release-0.250.rst

```diff
@@ -0,0 +1,24 @@
+=============
+Release 0.250
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a memory leak in `NodeTaskMap` which could lead to Full GC or OOMs in coordinator.
+* Improve Dictionary Block `getRegionLogicalSizeInBytes` performance.
+* Add DWRF Integer Dictionary encoding.
+* Add support for configuring the maximum number of unacknowledged source splits per task. This can be enabled by setting the ``max_unacknowledged_splits_per_task`` session property or ``node-scheduler.max-unacknowledged-splits-per-task`` configuration property.
+* Add 12 integral shift functions defined by Trino's BitwiseFunctions class to Presto's BitwiseFunctions class. These shift functions represent a complete set of shift operations on tinyints, smallints, integers and longs.
```

Comment:
```
Add bitwise shift functions :func:`func1`, :func:`func2` .... (:pr:`12345`)
```

Reply:
How about just Add bitwise shift functions (:pr:`12345`)?

Reply:
bitwise shift functions are not new - just new ones are added - so being explicit will help here.